app/controllers/api/events_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Api
+  # Controller to expose Events
+  class EventsController < ApplicationController
+    before_action :set_event, only: [:show]
+
+    # GET /api/event_dates/1
```

Comment:
This is bordering on nit-picky, but this comment should say `events` right?

Reply:
yes, I will update. thanks to pointing out that.